src/main/scala/info/bethard/timenorm/formal/Types.scala

```diff
@@ -31,14 +37,31 @@ object Modifier {
   */
 trait Period extends TimeExpression with TemporalAmount
 
-case class SimplePeriod(unit: TemporalUnit, n: Number, modifier: Modifier) extends Period {
-  override def addTo(temporal: Temporal): Temporal = ???
+case class SimplePeriod( unit: TemporalUnit, n: Number, modifier: Modifier) extends Period {
```

Comment:
No need to add a space here, and in general, for consistency, it would be better to not add spaces around parentheses (since the rest of the code doesn't).